src/Bridge/Symfony/Resources/config/services.yaml

```diff
@@ -0,0 +1,77 @@
+parameters:
+    default_elasticsearch_host: 'https://admin:admin@elasticsearch:9200'
```

Comment:
Package parameters should always be namespaced to avoid potential collisions with other things, for example `eonx_search_`